module/takeuntil/README.md

```diff
@@ -6,12 +6,12 @@ __Graph__
 ```
 a:               {---1---2---3---4}
 b:               {---------x------}
-takeUntil(a, b): {---1---2--------}
+takeUntil(b, a): {---1---2--------}
 ```
 
 __Signature__
 
-`Stream a -> Stream b -> Stream a`
+`Stream b -> Stream a -> Stream a`
```

Comment:
Maybe we should flip the meaning of `a` and `b` instead of flipping their order? As in `Stream a -> Stream b -> Stream b`?

Reply:
Ah! yes of course! that makes much more sense, nice catch!